### ISSUE TITLE
Disconnect in CNN-BiLSTM and XGBoost Model

### ISSUE THREAD

**Issue by zainabtareen** [zshicode/Attention-CLX-stock-prediction#3]
While running the Main.py, I saw an ambiguity in the code.

In PredictWithData func, there are these lines:
y_hat =  model.predict(testX)
testY, y_hat = xgb_scheduler(data_yuan, y_hat)

It looks like the xgboost model will take y_hat into account, but, upon inspection of xgb_scheduler, no such connection was found:

def xgb_scheduler(data,y_hat):
    close = data.pop('close')
    data.insert(5, 'close', close)
    train, test = prepare_data(data, n_test=len(y_hat), n_in=6, n_out=1)
    testY, y_hat2 = walk_forward_validation(train, test)
    return testY, y_hat2

(y_hat is just being used for calculating the length of test set)

The effect of y_hat is seen nowhere.?
